config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,11 @@
+CarrierWave.configure do |config|
+  config.storage = :fog
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['S3_KEY'],
+    aws_secret_access_key: ENV['S3_SECRET'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.